beacon_node/rest_api/tests/test.rs

```diff
@@ -338,11 +341,14 @@ fn validator_block_post() {
         .runtime()
         .block_on(remote_node.http.validator().publish_block(block.clone()))
         .expect("should publish block");
-    assert_eq!(
-        publish_status,
-        PublishStatus::Valid,
-        "the signed published block should be valid"
-    );
+
+    if cfg!(feature = "fake_crypto") {
```

Comment:
If the function is marked `ignore` when `fake_crypto` is enabled, then disabling the asserts is redundant. I think we should remove the function ignore attribute and just ignore the asserts (although I think `cfg!(not(feature = "fake_crypto"))` is probably what you meant)

Reply:
Hehe of course! My bad on this one, I thought I had already removed the function attribute... I didn't review my own PR, bad Scott!